Framework/PythonInterface/plugins/algorithms/ComputeCalibrationCoefVan.py

```diff
@@ -0,0 +1,168 @@
+from mantid.api import PythonAlgorithm, AlgorithmFactory, WorkspaceProperty, Progress, InstrumentValidator
+from mantid.kernel import Direction
+import mantid.simpleapi as api
+import numpy as np
+from scipy import integrate
+import scipy as sp
+import mlzutils
```

Comment:
I will refactor mlzutils. May be it would be better to submit a separate refactoring pull request, since several algorithms will get involved. However, do_gaussian_fit is also used in the other TOFTOF  algorithm which I am preparing for submission now. I've placed it in the mlzutils to avoid the code duplication. What would be better to do with do_gaussian_fit? May be also create a generic module?